samples/backups.js

```diff
@@ -0,0 +1,200 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Imports the Google Cloud client library
+const {Bigtable} = require('@google-cloud/bigtable');
+const uuid = require('uuid');
+const {inspect} = require('util');
+
+function generateBackupId() {
```

Comment:
These almost always belong in the samples test, and are passed in as a parameter to the main function (will keep reading though)